### ISSUE TITLE
@hansen7 @QingyongHu

### ISSUE THREAD

**Issue by soheilAppear** [QingyongHu/RandLA-Net#114]
@hansen7 @QingyongHu 

even iam trying to use pretrained model for S3DIS dataset

correct me if iam wrong
1) if iam using pretrained model then i dont have to run these lines
     sh jobs_6_fold_cv_s3dis.sh
     python utils/6_fold_cv.py
2) i just run    python utils/data_prepare_s3dis.py     and i succeded
 but when i run   python -B main_S3DIS.py --gpu 0 --mode test --test_area 1  
i got into  below result

---------------------------------------------------------------------------------------------------------
Area_1_office_31_KDTree.pkl 7.4 MB loaded in 0.0s
Area_1_hallway_4_KDTree.pkl 0.7 MB loaded in 0.0s
Area_1_office_15_KDTree.pkl 0.9 MB loaded in 0.0s

Preparing reprojected indices for testing
Area_1_hallway_2 done in 0.0s
Area_1_hallway_5 done in 0.0s
Area_1_office_16 done in 0.0s
Area_1_office_18 done in 0.0s
Area_1_office_28 done in 0.0s
Area_1_office_20 done in 0.0s
Area_1_office_22 done in 0.0s
Area_1_hallway_1 done in 0.0s
Area_1_office_1 done in 0.0s
Area_1_office_14 done in 0.0s
Area_1_office_30 done in 0.0s
Area_1_office_7 done in 0.0s
Area_1_office_3 done in 0.0s
Area_1_office_26 done in 0.0s
Area_1_office_13 done in 0.0s
Area_1_office_5 done in 0.0s
Area_1_pantry_1 done in 0.0s
Area_1_office_11 done in 0.0s
Area_1_office_2 done in 0.0s
Area_1_office_9 done in 0.0s
Area_1_office_21 done in 0.0s
Area_1_office_10 done in 0.0s
Area_1_hallway_8 done in 0.0s
Area_1_conferenceRoom_1 done in 0.0s
Area_1_conferenceRoom_2 done in 0.0s
Area_1_hallway_6 done in 0.1s
Area_1_office_19 done in 0.0s
Area_1_WC_1 done in 0.0s
Area_1_office_17 done in 0.0s
Area_1_hallway_3 done in 0.0s
Area_1_hallway_7 done in 0.0s
Area_1_office_12 done in 0.0s
Area_1_office_4 done in 0.0s
Area_1_office_29 done in 0.0s
Area_1_copyRoom_1 done in 0.0s
Area_1_office_23 done in 0.0s
Area_1_office_8 done in 0.0s
Area_1_office_25 done in 0.0s
Area_1_office_27 done in 0.0s
Area_1_office_6 done in 0.0s
Area_1_office_24 done in 0.0s
Area_1_office_31 done in 0.1s
Area_1_hallway_4 done in 0.0s
Area_1_office_15 done in 0.0s
Initiating input pipelines
WARNING:tensorflow:From /home/vikas/RandLA-Net/RandLANet.py:265: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
Instructions for updating:

Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

See `tf.nn.softmax_cross_entropy_with_logits_v2`.

/home/vikas/anaconda3/envs/randlanet/lib/python3.5/site-packages/tensorflow/python/ops/gradients_impl.py:108: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
results/Log_2020-06-22_11-52-29/snapshots
Model restored from results/Log_2020-06-22_11-52-29/snapshots/snap-26001
Killed
---------------------------------------------------------------------------------------------------

process got killed 
let me know anyone can help

_Originally posted by @v-shetty in https://github.com/QingyongHu/RandLA-Net/issues/40#issuecomment-648125788_

**Comment by soheilAppear** [comment]
I have the same issue and the process gets killed when I want to run inference based on the pre-trained model

**Comment by soheilAppear** [comment]
I found the answer for that. If somebody have the same issue, try to reduce the batch size by going to https://github.com/QingyongHu/RandLA-Net/blob/3acc85ded087da7134f296b384ddbed9d1438d5a/helper_tool.py#L25-L26

A similar question was answered here:
https://github.com/QingyongHu/RandLA-Net/issues/59
